account_analytic_no_lines/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2016 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Account Analytic No Lines',
+    'summary': """
+        Hide analytics lines and disable
+        their generation from a move line.""",
+    'version': '12.0.1.0.0',
+    'license': 'AGPL-3',
+    'author': 'ACSONE SA/NV,Odoo Community Association (OCA)',
+    'website': 'https://acsone.eu/',
```

Comment:
@ThomasBinsfeld Replace URL as per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L10)